addons/sale_management/models/sale_order_template.py

```diff
@@ -29,6 +29,14 @@ def _get_default_require_payment(self):
         domain=[('model', '=', 'sale.order')],
         help="This e-mail template will be sent on confirmation. Leave empty to send nothing.")
     active = fields.Boolean(default=True, help="If unchecked, it will allow you to hide the quotation template without removing it.")
+    company_id = fields.Many2one('res.company', string='Company')
+
+    @api.constrains('company_id', 'sale_order_template_line_ids')
```

Comment:
```suggestion
    @api.constrains('company_id', 'sale_order_template_line_ids', 'sale_order_template_option_ids')
```
We have to check this for all products of `sale_order_template_option_ids`.

See on: http://551261-34342-0b6909.runbot22.odoo.com/web?debug=1#id=1&action=892&model=sale.order.template&view_type=form&cids=23%2C1&menu_id=569

![screenshot-551261-34342-0b6909 runbot22 odoo com-2019 06 28-23-17-52](https://user-images.githubusercontent.com/16624719/60361161-23531800-99fb-11e9-940b-587cc72588fe.jpg)

Reply:
This will be done ;)